### PR TITLE
TNFT send strings

### PR DESCRIPTION
**Pull request by vetkamak**

### CODE REVIEW

File: src/ko.json [hunk 1]
```diff
@@ -85,9 +85,12 @@
...
             },
             "send": {
-                "confirm": "확인",
+                "confirmTitle": "Confirm sending",
+                "continueBtn": "Continue",
+                "fee": "Network fees",
                 "receiver": "수신인",
-                "title": "보내기"
+                "sendBtn": "Send",
+                "sendTitle": "Send NFT"
```
Comment (sertony) [inline src/ko.json]: Consider translating. @kkkhesha FYI

File: src/ru.json [hunk 2]
```diff
@@ -84,9 +84,12 @@
...
             },
             "send": {
-                "confirm": "Подтвердить",
+                "confirmTitle": "Confirm sending",
+                "continueBtn": "Continue",
+                "fee": "Network fees",
                 "receiver": "Кому",
-                "title": "Отправка"
+                "sendBtn": "Send",
+                "sendTitle": "Send NFT"
```
Comment (sertony) [inline src/ru.json]: Consider translating. @AnnStepanova FYI

File: src/ko.json [hunk 3]
```diff
@@ -85,9 +85,12 @@
                 "trending": "유행"
             },
             "send": {
-                "confirm": "확인",
+                "confirmTitle": "Confirm sending",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "confirmTitle": "전송 확인",
```

File: src/ko.json [hunk 4]
```diff
@@ -85,9 +85,12 @@
...
+                "continueBtn": "Continue",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "continueBtn": "계속",
```

File: src/ko.json [hunk 5]
```diff
@@ -85,9 +85,12 @@
...
+                "fee": "Network fees",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "fee": "네트워크 수수료",
```

File: src/ko.json [hunk 6]
```diff
@@ -85,9 +85,12 @@
...
                 "receiver": "수신인",
-                "title": "보내기"
+                "sendBtn": "Send",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "sendBtn": "전송",
```

File: src/ko.json (hunk 1 above)
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "sendTitle": "NFT 전송"
```

Comment (kkkhesha) [reply src/ko.json]: done

File: src/ru.json [hunk 7]
```diff
@@ -84,9 +84,12 @@
                 "trending": "Тренды"
             },
             "send": {
-                "confirm": "Подтвердить",
+                "confirmTitle": "Confirm sending",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "confirmTitle": "Подтверждение",
```

File: src/ru.json [hunk 8]
```diff
@@ -84,9 +84,12 @@
...
+                "continueBtn": "Continue",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "continueBtn": "Продолжить",
```

File: src/ru.json [hunk 9]
```diff
@@ -84,9 +84,12 @@
...
+                "fee": "Network fees",
                 "receiver": "Кому",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "receiver": "Получатель",
```

File: src/ru.json [hunk 10]
```diff
@@ -84,9 +84,12 @@
...
-                "title": "Отправка"
+                "sendBtn": "Send",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "sendBtn": "Отправить",
```

File: src/ru.json [hunk 11]
```diff
@@ -84,9 +84,12 @@
                 "trending": "Тренды"
             },
             "send": {
-                "confirm": "Подтвердить",
+                "confirmTitle": "Confirm sending",
+                "continueBtn": "Continue",
+                "fee": "Network fees",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "fee": "Комиссия",
```

**Review by sertony — APPROVED** [review]

**Review by joydark — APPROVED** [review]

**Pull request merged**